ui/app/components/splash-page.hbs

```diff
@@ -9,12 +9,9 @@
       <div class="splash-page-header" data-test-splash-page-header>
         {{yield to="header"}}
       </div>
-      <div class="splash-page-sub-header" data-test-splash-page-sub-header>
-        {{yield to="subHeader"}}
-      </div>
-      <div class="login-form box is-paddingless is-relative" data-test-splash-page-content>
```

Comment:
If I recall this was to accomodate something with MFA? Did it move to the component?

Reply:
Good question! The `subHeader` is where the [namespace input rendered](https://github.com/hashicorp/vault/pull/30521/files#diff-2279917db00e6f8bab9b2f2ae39bc24442775b61fe09cca4a9853ef798010b96L42), which now renders in the `Auth::FormTemplate`

https://github.com/hashicorp/vault/blob/e77f206b41bb2f70f691085f04af5da54582f006/ui/app/components/auth/form-template.hbs#L18-L23

And then the content `div`on 15 is just replaced with the `Hds::Card::Container` to update the css with the rounded edges. This removes the need for the `login-form` css class and replaces it with `overflow-auto` class.

Reply:
Ahh yes that's right. I must have been thinking of something else related to MFA. Nice to remove that block since it was only used for the one case!

Reply:
There's also the mfa setup steps [here](https://github.com/hashicorp/vault/pull/30521/files#diff-52c7a6f53b3e24a3ade5c66b3abe395ee24fd7b19e74bf97ebe315683df56e0c) - I wonder if that's what you were thinking of 🤔 Hopefully I didn't miss anything 😅